src/lib.rs

```diff
@@ -237,6 +237,10 @@
 //! have a size of 32 vectors (on ARMv6-M) or 240 vectors (on ARMv7-M). This array is located after
 //! `__EXCEPTIONS` in the `.vector_table` section.
 //!
+//! - `__pre_init`. This is a function to be run before RAM is initialized. It defaults pointing at
+//! `0` and if not changed to point to another address, usually by calling the `pre_init!` macro,
```

Comment:
There is an error here, should be 1

Reply:
Ah, I knew I should have looked at the docs once again!

Reply:
It would be useful to include a short comment about why it can't be 0 too.